rts/Game/Camera/SpringController.cpp

```diff
@@ -22,6 +22,7 @@ CONFIG(bool,  CamSpringLockCardinalDirections).defaultValue(true).description("W
 CONFIG(bool,  CamSpringZoomInToMousePos).defaultValue(true);
 CONFIG(bool,  CamSpringZoomOutFromMousePos).defaultValue(false);
 CONFIG(bool,  CamSpringEdgeRotate).defaultValue(false).description("Rotate camera when cursor touches screen borders.");
+CONFIG(float, CameraMoveFastMult).defaultValue(3.0f).minimumValue(1.0f).description("The multiplier applied to speed when camera is in movefast state.");
```

Comment:
Please extract the config definition and listening to `Camera` or `CameraController` so it's defined in only one place.